File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Clis/CLIS2015U02GrdStatusInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Clis
{
    [Serializable]
    public class CLIS2015U02GrdStatusInfo
    {
        private String _protocolId;
        private String _statusId;
        private String _statusCode;
        private String _statusName;
        private String _sortNo;
        private String _displayFlg;
        private String _rowState;

        public String ProtocolId
        {
            get { return this._protocolId; }
            set { this._protocolId = value; }
        }

        public String StatusId
        {
            get { return this._statusId; }
            set { this._statusId = value; }
        }

        public String StatusCode
        {
            get { return this._statusCode; }
            set { this._statusCode = value; }
        }

        public String StatusName
        {
            get { return this._statusName; }
            set { this._statusName = value; }
        }

        public String SortNo
        {
            get { return this._sortNo; }
            set { this._sortNo = value; }
        }

        public String DisplayFlg
        {
            get { return this._displayFlg; }
            set { this._displayFlg = value; }
        }

        public String RowState
        {
            get { return this._rowState; }
            set { this._rowState = value; }
        }

        public CLIS2015U02GrdStatusInfo() { }

        public CLIS2015U02GrdStatusInfo(String protocolId, String statusId, String statusCode, String statusName, String sortNo, String displayFlg, String rowState)
        {
            this._protocolId = protocolId;
            this._statusId = statusId;
            this._statusCode = statusCode;
            this._statusName = statusName;
            this._sortNo = sortNo;
            this._displayFlg = displayFlg;
            this._rowState = rowState;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/clis/CLIS2015U02GrdStatusInfo.java
package nta.med.data.model.ihis.clis;


public class CLIS2015U02GrdStatusInfo {
	  private Integer protocolId;
      private Integer statusId;
      private String statusCode;
      private String statusName;
      private Integer sortNo;
      private String displayFlg;
	public CLIS2015U02GrdStatusInfo(Integer protocolId, Integer statusId,
			String statusCode, String statusName, Integer sortNo,
			String displayFlg) {
		super();
		this.protocolId = protocolId;
		this.statusId = statusId;
		this.statusCode = statusCode;
		this.statusName = statusName;
		this.sortNo = sortNo;
		this.displayFlg = displayFlg;
	}
	public Integer getProtocolId() {
		return protocolId;
	}
	public void setProtocolId(Integer protocolId) {
		this.protocolId = protocolId;
	}
	public Integer getStatusId() {
		return statusId;
	}
	public void setStatusId(Integer statusId) {
		this.statusId = statusId;
	}
	public String getStatusCode() {
		return statusCode;
	}
	public void setStatusCode(String statusCode) {
		this.statusCode = statusCode;
	}
	public String getStatusName() {
		return statusName;
	}
	public void setStatusName(String statusName) {
		this.statusName = statusName;
	}
	public Integer getSortNo() {
		return sortNo;
	}
	public void setSortNo(Integer sortNo) {
		this.sortNo = sortNo;
	}
	public String getDisplayFlg() {
		return displayFlg;
	}
	public void setDisplayFlg(String displayFlg) {
		this.displayFlg = displayFlg;
	}
}
